docs/releases/1.10.txt

```diff
@@ -353,7 +353,11 @@ Backwards incompatible changes in 1.10
 Database backend API
 --------------------
 
-* ...
+* The GIS field ``AreaField`` uses an unspecified underlying numeric type,
```

Comment:
I think there may be some GIS around the Mongo-db nonrel backend.
I don't think any backend _needs_ to adapt to this change, but a hypothetical backend might benefit from adaptng to it (e.g. if there is a choice to be made about the type of data returned)